src/components/SingleArticle.jsx

```diff
@@ -22,27 +25,47 @@ function SingleArticle({isLoading, setIsLoading}) {
             })
     }, [article_id, setIsLoading])
 
+    const updateVoteNum = (e) => {
+        const newVotes = +e.target.value;
+        setVoteError("");
+        setVotes((currVote) => {
+            return currVote + newVotes;
+        })
+        updateVotes(article_id, newVotes).catch((err) => {
+            setVotes((currVote) => {
+                return currVote - newVotes;
+            })
+            setVoteError("Oops - something went wrong!")
+        })
+    }
+
     if (isLoading) {
         return (
-            <img src="https://img.pikbest.com/png-images/20190918/cartoon-snail-loading-loading-gif-animation_2734139.png!bw700" alt="loading" id="loading-img"/>
+            <img src="https://img.pikbest.com/png-images/20190918/cartoon-snail-loading-loading-gif-animation_2734139.png!bw700" alt="loading" id="loading-img" />
         )
     }
 
     return (
         <section>
-        <article className="single-article">
-            <h2>{singleArticle.title}</h2>
-            <div className="single-article-details">
-                <img id="author-avatar" src={author.avatar_url} alt={singleArticle.author} />
-                <p>{singleArticle.author}</p>
-                <p className="topic-label">{singleArticle.topic}</p>
-                <p>{singleArticle.created_at}</p>
-            </div>
-            <img src={singleArticle.article_img_url} alt={singleArticle.title} />
-            <p className="article-body">{singleArticle.body}</p>
-            <br></br>
-        </article>
-        <Comments/>
+            <article className="single-article">
+                <h2>{singleArticle.title}</h2>
+                <div className="single-article-details">
+                    <img id="author-avatar" src={author.avatar_url} alt={singleArticle.author} />
+                    <p>{singleArticle.author}</p>
+                    <p className="topic-label">{singleArticle.topic}</p>
+                    <p>{singleArticle.created_at}</p>
+                </div>
+                <img src={singleArticle.article_img_url} alt={singleArticle.title} />
```

Comment:
You could make these things loads DRY-er by de-structuring the values from the objects before the return!